ocular/src/chain/client/tx.rs

```diff
@@ -98,71 +164,10 @@ impl ChainClient {
         self.sign_and_send_msg(
             sender_account.public_key,
             sender_account.private_key,
-            amount,
-            tx_body,
-            tx_metadata,
-        )
-        .await
-    }
-
-    /// Sign and send delegate message
-    pub async fn delegate(
-        &self,
-        delegator_account: Account,
-        validator_account_id: AccountId,
-        amount: Coin,
-        tx_metadata: TxMetadata,
-    ) -> Result<Response, TxError> {
-        // Create delegate message for amount
-        let msg = MsgDelegate {
-            delegator_address: delegator_account.id,
-            validator_address: validator_account_id,
-            amount: amount.clone(),
-        };
-
-        // Build tx body.
-        let tx_body = match msg.to_any() {
-            Ok(msg) => tx::Body::new(vec![msg], &tx_metadata.memo, tx_metadata.timeout_height),
-            Err(err) => return Err(TxError::SerializationError(err.to_string())),
-        };
-
-        self.sign_and_send_msg(
-            delegator_account.public_key,
-            delegator_account.private_key,
-            amount,
-            tx_body,
-            tx_metadata,
-        )
-        .await
-    }
-
-    /// Sign and send undelegate message
-    pub async fn undelegate(
-        &self,
-        delegator_account: Account,
-        validator_account_id: AccountId,
-        amount: Coin,
-        tx_metadata: TxMetadata,
-    ) -> Result<Response, TxError> {
-        // Create undelegate message for amount
-        let msg = MsgUndelegate {
-            delegator_address: delegator_account.id,
-            validator_address: validator_account_id,
-            amount: amount.clone(),
-        };
-
-        // Build tx body.
-        let tx_body = match msg.to_any() {
-            Ok(msg) => tx::Body::new(vec![msg], &tx_metadata.memo, tx_metadata.timeout_height),
-            Err(err) => return Err(TxError::SerializationError(err.to_string())),
-        };
-
-        self.sign_and_send_msg(
-            delegator_account.public_key,
-            delegator_account.private_key,
-            amount,
```

Comment:
This is just a clean removal of delegate and undelegate msg types. Discovered some bugs with this while testing and decided to remove and bring back in fixed state later.